api/v1/webspherelibertyapplication_types.go

```diff
@@ -172,13 +173,12 @@ type WebSphereLibertyApplicationSpec struct {
 	DisableServiceLinks *bool `json:"disableServiceLinks,omitempty"`
 }
 
-// License information is required.
 type License struct {
 	// Product edition. Defaults to IBM WebSphere Application Server. Other options: IBM WebSphere Application Server Liberty Core, IBM WebSphere Application Server Network Deployment
 	// +operator-sdk:csv:customresourcedefinitions:order=100,type=spec,displayName="Edition"
 	Edition LicenseEdition `json:"edition,omitempty"`
 
-	// Entitlement source for the product. Defaults to Standalone. Other options: IBM Cloud Pak for Applications, IBM WebSphere Application Server Family Edition, IBM WebSphere Hybrid Edition
+	// Entitlement source for the product. Defaults to Standalone. Other options: IBM Cloud Pak for Applications Advanced, IBM Cloud Pak for Applications Standard, IBM WebSphere Hybrid Edition, IBM WebSphere Application Server Family Edition
```

Comment:
The description no longer mentions "IBM Cloud Pak for Applications", but that is still a valid option in the enum. Should the description still mention this option?

Reply:
Updated the description to mention that the "IBM Cloud Pak for Applications" option is deprecated. Thanks